contrib/graphql-codegen-client-preset/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@swc-contrib/plugin-graphql-codegen-client-preset",
+  "version": "0.0.0",
```

Comment:
I'm not sure if version here should be the continuation of the existing package? Or should it start again? 🤔

Reply:
I'll follow the existing package